Makefile

```diff
@@ -290,7 +290,7 @@ serve-docs:
 # tagged version. Once the unstaged changes are ready, they can be added to the
 # staging area using 'git add' and then committed.
 #
-# To run gen-docs: "NEW_DOCS_VERSION=v1.1.0 make gen-docs"
+# To run gen-docs: "NEW_DOCS_VERSION=v1.4 VELERO_VERSION=v1.4.0 make gen-docs"
```

Comment:
This PR is to allow us to reuse the same docs version across patch releases, correct?
In which case, I think we should be updating the `gen-docs.sh` at https://github.com/vmware-tanzu/velero/pull/2473/files#diff-5f45d33e4fc9e049e1b0beee45db8269R25, no?

Reply:
This link should be better https://github.com/vmware-tanzu/velero/blob/master/hack/gen-docs.sh#L25

Reply:
Well, you shouldn't really have to run the gen-docs process more than once per major/minor -- any "bug fixes" to the docs accompanying a patch release would be made manually in the major/minor.  Additionally, you could run into issues where docs changes made in master that are intended to be for the next feature release would get automatically pulled into a patch release's docs.

Reply:
That sounds like what we've been doing in practice and also reasonable to continue. So 👍